patches/react-native-reanimated+2.17.0.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/node_modules/react-native-reanimated/react-native-reanimated.d.ts b/node_modules/react-native-reanimated/react-native-reanimated.d.ts
+index c33d830..75f094f 100644
+--- a/node_modules/react-native-reanimated/react-native-reanimated.d.ts
++++ b/node_modules/react-native-reanimated/react-native-reanimated.d.ts
+@@ -129,6 +129,7 @@ declare module 'react-native-reanimated' {
+     export type TransformStyleTypes = TransformsStyle['transform'] extends
+       | readonly (infer T)[]
+       | undefined
++      | string
```

Comment:
For context on this change: https://github.com/software-mansion/react-native-reanimated/issues/4548#issuecomment-1581982288

Maybe I should stick that link into the commit message.